legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/test/runner/service/ServiceTestRunner.java

```diff
@@ -273,9 +275,13 @@ private RichServiceTestResult executeTestAsserts(SingleExecutionPlan executionPl
                     {
                         parameters = ListIterate.zip(((PureExecution) this.service.execution).func.parameters, tc.getOne().parametersValues).toMap(
                                 p -> p.getOne().name,
-                                p -> new ConstantResult(p.getTwo() instanceof PureList
-                                        ? ListIterate.collect(((PureList) p.getTwo()).values, v -> v.accept(DEPRECATED_PureGrammarComposerCore.Builder.newInstance().withValueSpecificationAsExternalParameter().build()))
-                                        : p.getTwo().accept(DEPRECATED_PureGrammarComposerCore.Builder.newInstance().withValueSpecificationAsExternalParameter().build())));
+                                p -> p.getTwo() instanceof Collection
```

Comment:
I assume 
p.getTwo() instanceof Collection -> is for studio flow
p.getTwo() instanceof PureList     -> is for pure ide flow

show we add a comment so that we know the reason?

Reply:
Comments added